Client/core/CMainMenu.cpp

```diff
@@ -221,6 +224,53 @@ CMainMenu::CMainMenu(CGUI* pManager)
     m_pLatestNews->SetProperty("InheritsAlpha", "False");
     m_pLatestNews->SetVisible(false);
 
+    static const std::array<std::pair<const char*, const char*>, 6> m_socialMediaLinks{{
+        {"website", "https://multitheftauto.com"},
+        {"discord", "https://discord.gg/mtasa"},
```

Comment:
```suggestion
        {"discord", "https://discord.com/invite/mtasa"},
```

Reply:
What if your invite expires?
The botder idea of forwarding this from the MTA web server is safer in my opinion.

Reply:
Then we update it. It won't expire out of the blue.

Reply:
> What if your invite expires?
> The botder idea of forwarding this from the MTA web server is safer in my opinion.

In general it's good practice to keep links in one place so you only have to update once. Afaik there is already mtasa.com/discord so it can be put here, no?

Reply:
I'm part of the team that has to maintain this stuff, it's just not a good thing. There is very little chance these links would ever change, and we have full control over the main menu.

Reply:
Wouldn't be better to don't use a custom invite link once we would be relying on the server's boosts and they might expire or people might not want to boost again?

Reply:
We are not relying on server boosts, Discord has granted us permanent vanity url as per their open source program.

Reply:
Very cool then sorry didn't know. Makes sense to use discord.com/invite/mtasa here then imo.